SS_Activation_OpenMC.py

```diff
@@ -0,0 +1,150 @@
+#Depletion calculation
+W.depletable = True
+W.volume = 4.0/3.0 * np.pi * (R_2**3 - R_1**3) #volume of W wall material
+operator = openmc.deplete.CoupledOperator(model, normalization_mode='source-rate')
+time_steps = [3e8, 86400, 2.6e6]
+source_rates = [1E+18, 0, 0]
+integrator = openmc.deplete.PredictorIntegrator(operator=operator, timesteps=time_steps, source_rates=source_rates, timestep_units='s')
+integrator.integrate()
+
+# Depletion results file
+results = openmc.deplete.Results(filename='depletion_results.h5')
+# Obtain nuclide concentration as a function of time
+
+time, n_He4 = results.get_atoms('1', 'He4', nuc_units = 'atom/cm3')
+time, n_W179 = results.get_atoms('1', 'W179', nuc_units = 'atom/cm3')
+time, n_W180 = results.get_atoms('1', 'W180', nuc_units = 'atom/cm3')
+time, n_W181 = results.get_atoms('1', 'W181', nuc_units = 'atom/cm3')
+time, n_W182 = results.get_atoms('1', 'W182', nuc_units = 'atom/cm3')
+time, n_W183 = results.get_atoms('1', 'W183', nuc_units = 'atom/cm3')
+time, n_W184 = results.get_atoms('1', 'W184', nuc_units = 'atom/cm3')
+time, n_W185 = results.get_atoms('1', 'W185', nuc_units = 'atom/cm3')
+time, n_W186 = results.get_atoms('1', 'W186', nuc_units = 'atom/cm3')
+time, n_W187 = results.get_atoms('1', 'W187', nuc_units = 'atom/cm3')
+time, n_Re185 = results.get_atoms('1', 'Re185', nuc_units = 'atom/cm3')
+time, n_Re187 = results.get_atoms('1', 'Re187', nuc_units = 'atom/cm3')
+time, n_Os187 = results.get_atoms('1', 'Os187', nuc_units = 'atom/cm3')
+
+print(time, n_He4)
+print(time, n_W179)
+print(time, n_W180)
+print(time, n_W181)
+print(time, n_W182)
+print(time, n_W183)
+print(time, n_W184)
+print(time, n_W185)
+print(time, n_W186)
+print(time, n_W187)
+print(time, n_Re185)
+print(time, n_Re187)
+print(time, n_Os187)
+
+# Plotting the number densities of each nuclide:
+plt.plot(time, n_He4, marker='.', linestyle='solid', color='red', label='He4')
+plt.plot(time, n_W179, marker='.', linestyle='solid', color='blue', label="W179")
+#plt.plot(time, n_W180, marker='.', linestyle='solid', color='green', label="W180")
+#plt.plot(time, n_W181, marker='.', linestyle='solid', color='deepskyblue', label="W181")
+#plt.plot(time, n_W182, marker='.', linestyle='solid', color='peru', label="W182")
+#plt.plot(time, n_W183, marker='.', linestyle='solid', color='plum', label="W183")
+#plt.plot(time, n_W184, marker='.', linestyle='solid', color='crimson', label="W184")
+plt.plot(time, n_W185, marker='.', linestyle='solid', color='rosybrown', label="W185")
+#plt.plot(time, n_W186, marker='.', linestyle='solid', color='gray', label="W186")
+plt.plot(time, n_W187, marker='.', linestyle='solid', color='lime', label="W187")
+plt.plot(time, n_Re185, marker='.', linestyle='solid', color='gold', label="Re185")
+plt.plot(time, n_Re187, marker='.', linestyle='solid', color='tomato', label="Re187")
+plt.plot(time, n_Os187, marker='.', linestyle='solid', color='cyan', label="Os187")
```

Comment:
Some kind of list data structure may be better?
```suggestion
num_dens = {}
plot_colors = {'He4' : 'red', 'W179' : 'green', 'W180' : blue } #etcetera

for nuclide, plot_color in plot_colors.items():
    time, num_dens[nuclide] =results.get_atoms('1', nuclide, nuc_units = 'atom/cm3')
    print(time, num_dens[nuclide])
    plt.plot(time, num_dens[nuclide], marker='.', linestyle='solid', color=plot_color, label=nuclide)
```

Reply:
Missed this comment -- will add it to the code